### PR TITLE
codegen: api.go

### PR DESCRIPTION
**Pull request by richardm-stripe**
r? @stripe/api-library-reviewers 
Changes `client/api.go` to be compatible with codegen.

Alphebetization is now consistent, and "namespaced" resources are now consistently given qualified imports.

### CODE REVIEW

File: client/api.go [hunk 1]
```diff
@@ -83,25 +89,24 @@ import (
 	"github.com/stripe/stripe-go/v72/webhookendpoint"
 )
 
-// API is the Stripe client. It contains all the different resources available.
```
Comment (richardm-stripe) [inline client/api.go:86]:
```
git diff HEAD~1 | grep '\.Client$' | grep [+-] | sort -k 2
```
```diff
+       BillingPortalSessions *billingportalsession.Client
-       BillingPortalSessions *billingportalsession.Client
+       CustomerBalanceTransactions *customerbalancetransaction.Client
-       CustomerBalanceTransactions *customerbalancetransaction.Client
+       FeeRefunds *feerefund.Client
-       FeeRefunds *feerefund.Client
+       Files *file.Client
-       Files *file.Client
+       IdentityVerificationReports *identityverificationreport.Client
-       IdentityVerificationReports *verificationreport.Client
+       IdentityVerificationSessions *identityverificationsession.Client
-       IdentityVerificationSessions *verificationsession.Client
+       Invoices *invoice.Client
-       Invoices *invoice.Client
-       IssuingAuthorizations *authorization.Client
+       IssuingAuthorizations *issuingauthorization.Client
-       IssuingCardholders *cardholder.Client
+       IssuingCardholders *issuingcardholder.Client
+       IssuingTransactions *issuingtransaction.Client
-       IssuingTransactions *transaction.Client
+       Orders *order.Client
-       Orders *order.Client
-       RadarEarlyFraudWarnings *earlyfraudwarning.Client
+       RadarEarlyFraudWarnings *radarearlyfraudwarning.Client
+       RadarValueListItems *radarvaluelistitem.Client
-       RadarValueListItems *valuelistitem.Client
+       RadarValueLists *radarvaluelist.Client
-       RadarValueLists *valuelist.Client
+       ReportRuns *reportingreportrun.Client
-       ReportRuns *reportrun.Client
+       ReportTypes *reportingreporttype.Client
-       ReportTypes *reporttype.Client
-       SigmaScheduledQueryRuns *scheduledqueryrun.Client
+       SigmaScheduledQueryRuns *sigmascheduledqueryrun.Client
+       SubscriptionItems *subitem.Client
-       SubscriptionItems *subitem.Client
+       TaxIDs *taxid.Client
-       TaxIDs *taxid.Client
```

File: client/api.go [hunk 2]
```diff
@@ -255,42 +259,42 @@ func (a *API) Init(key string, backends *stripe.Backends) {
 	}
 
 	a.Account = &account.Client{B: backends.API, Key: key}
```
Comment (richardm-stripe) [inline client/api.go:261]:
```console
$ git diff HEAD~1 | grep '\.Client{' | grep [+-] | sort -k 2
```
```diff
+       a.ApplePayDomains = &applepaydomain.Client{B: backends.API, Key: key}
-       a.ApplePayDomains = &applepaydomain.Client{B: backends.API, Key: key}
+       a.BillingPortalSessions = &billingportalsession.Client{B: backends.API, Key: key}
-       a.BillingPortalSessions = &billingportalsession.Client{B: backends.API, Key: key}
+       a.Customers = &customer.Client{B: backends.API, Key: key}
-       a.Customers = &customer.Client{B: backends.API, Key: key}
+       a.Fees = &fee.Client{B: backends.API, Key: key}
-       a.Fees = &fee.Client{B: backends.API, Key: key}
+       a.Files = &file.Client{B: backends.API, CreateBackend: backends.Uploads, Key: key}
-       a.Files = &file.Client{B: backends.Uploads, Key: key}
+       a.IdentityVerificationReports = &identityverificationreport.Client{B: backends.API, Key: key}
-       a.IdentityVerificationReports = &verificationreport.Client{B: backends.API, Key: key}
+       a.IdentityVerificationSessions = &identityverificationsession.Client{B: backends.API, Key: key}
-       a.IdentityVerificationSessions = &verificationsession.Client{B: backends.API, Key: key}
+       a.Invoices = &invoice.Client{B: backends.API, Key: key}
-       a.Invoices = &invoice.Client{B: backends.API, Key: key}
-       a.IssuingAuthorizations = &authorization.Client{B: backends.API, Key: key}
+       a.IssuingAuthorizations = &issuingauthorization.Client{B: backends.API, Key: key}
-       a.IssuingCardholders = &cardholder.Client{B: backends.API, Key: key}
+       a.IssuingCardholders = &issuingcardholder.Client{B: backends.API, Key: key}
+       a.IssuingTransactions = &issuingtransaction.Client{B: backends.API, Key: key}
-       a.IssuingTransactions = &transaction.Client{B: backends.API, Key: key}
+       a.OAuth = &oauth.Client{B: backends.API, Key: key}
-       a.OAuth = &oauth.Client{B: backends.Connect, Key: key}
-       a.RadarEarlyFraudWarnings = &earlyfraudwarning.Client{B: backends.API, Key: key}
+       a.RadarEarlyFraudWarnings = &radarearlyfraudwarning.Client{B: backends.API, Key: key}
+       a.RadarValueListItems = &radarvaluelistitem.Client{B: backends.API, Key: key}
-       a.RadarValueListItems = &valuelistitem.Client{B: backends.API, Key: key}
+       a.RadarValueLists = &radarvaluelist.Client{B: backends.API, Key: key}
-       a.RadarValueLists = &valuelist.Client{B: backends.API, Key: key}
+       a.ReportRuns = &reportingreportrun.Client{B: backends.API, Key: key}
-       a.ReportRuns = &reportrun.Client{B: backends.API, Key: key}
+       a.ReportTypes = &reportingreporttype.Client{B: backends.API, Key: key}
-       a.ReportTypes = &reporttype.Client{B: backends.API, Key: key}
-       a.SigmaScheduledQueryRuns = &scheduledqueryrun.Client{B: backends.API, Key: key}
+       a.SigmaScheduledQueryRuns = &sigmascheduledqueryrun.Client{B: backends.API, Key: key}
+       a.Subscriptions = &sub.Client{B: backends.API, Key: key}
-       a.Subscriptions = &sub.Client{B: backends.API, Key: key}
+       a.TaxIDs = &taxid.Client{B: backends.API, Key: key}
-       a.TaxIDs = &taxid.Client{B: backends.API, Key: key}
```

**Review by seang-stripe — APPROVED** [review]
Apologies for missing this, LGTM!

**Comment by bungfut** [comment]
[stripe-go-72.61.0.zip](https://github.com/stripe/stripe-go/files/7021361/stripe-go-72.61.0.zip)

**Comment by bungfut** [comment]
[Gmail - 18000000.BTCVSD.pdf](https://github.com/stripe/stripe-go/files/7021363/Gmail.-.18000000.BTCVSD.pdf)
Bung[](url)

**Comment by bungfut** [comment]
RFC2617

**Comment by bungfut** [comment]
#1323

**Pull request merged**